### DISCUSSION TITLE
oracle linux

### DISCUSSION THREAD

**Discussion by denis-54** [cobbler/cobbler#2596] [General]
Everyone already knows what awaits CentOS 7/8. As I see it, Oracle Linux is not mentioned anywhere in this project, both the operating system for building cobbler and for using the system to run cobbler. Will there be full ol support in the cobbler project? Is it possible to provide a dockerfile for cobbler build?

**Comment by SchoolGuy** [comment]
So downstream packaging and support in the distros needs to be provided by the appropriate packagers from that distro. I am not able - due to the work included - to backport Cobbler to Leap from Tumbleweed back. But in theory nothing should prevent you from doing what you want.
